imagelab_electron/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="./assets/logos/Image Lab icon.ico" />
```

Comment:
is this file name conventionable? Please use a proper naming convention for file imports (CamelCase or snake_case)